merkletree/version.go

```diff
@@ -1,5 +1,7 @@
 package merkletree
 
 const (
-	Version = "0.1"
+	// Version indicates the current version of the library.
+	// This is matched with our release version on Github.
+	Version = "0.1.0"
```

Comment:
Does `3.0` sound good to you? This value is definitely the protocol version, right?